reports/blueprint/report.tex

```diff
@@ -45,26 +45,38 @@
 \end{titlepage}
 \section*{Executive Summary}
 % pop a one pager in here. Ideally less
-
+% Timothy
 \newpage
 
 \tableofcontents
 \listoffigures
 \listoftables
 \newpage
 \setstretch{1.5}
-\section{Introduction} \label{sec:intro} % John
-% As short as possible. Total BS LMAO
-Welcome to this 1.5 spaced introduction. it should be 1.5 spaced. See the
-setstretch command shown above in the tex file.
+\section{Introduction} \label{sec:intro} % Ethan
+This blueprint outlines Group 18's capstone project TeachEE, which was
+originally proposed in ELEC390 \cite{prop_390}. This report is intended the ELEC 498 instructors
+and Group 18's supervisors. The report provides the baseline requirements for
+TeachEE. These requirements should be treated as the minimum performance
+required for the showcase of the project in March. This blueprint consists of a
+work breakdown structure, progress up to the time of writing this document,
+budget, and strategies to mitigate issues. Appendices are
+included for relevant technical information that does not fit in the main body
```

Comment:
Better to say it includes info that supplements the report

Reply:
Fixed